src/ResourceManager/ServiceBus/Commands.ServiceBus/Models/TopicAttributes.cs

```diff
@@ -182,31 +173,6 @@ public TopicAttributes(SBTopic topicResource)
         /// </summary>
         [ObsoleteAttribute("'IsExpress' property is mark as obsolete and will be removed in upcoming breaking changes build", false)]
         public bool? IsExpress { get; set; }
-
-        /// <summary>
```

Comment:
Did you mean to leave the Location property?

Reply:
my bad I missed,  removed the properties marked as obsolete.